bindings/java/src/test/java/org/openucx/jucx/UcpListenerTest.java

```diff
@@ -126,6 +126,17 @@ public void testConnectionHandler() {
 
         UcpRequest sent = serverToClient.sendStreamNonBlocking(
             ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE), null);
+
+        // Progress all workers to make sure recv request will complete immediately
+        for (int i = 0; i < 10; i++) {
```

Comment:
why 10 iterations? :)

Reply:
Need to force request to complete send and arrive to receive queue, to test immidiate completion, when receive request will be submit. 10 just to make sure it will complete these operations on all test platforms. Could be less, but we use 10 iterations on connection establishment: 
https://github.com/openucx/ucx/blob/master/bindings/java/src/main/java/org/openucx/jucx/examples/UcxReadBWBenchmarkReceiver.java#L45